src/pages/admin/pages/faq/FaqPage.vue

```diff
@@ -3,16 +3,18 @@
     <va-card class="mb-8">
       <va-card-title> Do you have any questions? </va-card-title>
       <va-card-content>
-        <va-input class="m-3">
+        <va-input v-model="searchValue" class="m-3">
           <template #prepend>
             <va-icon name="search" />
           </template>
         </va-input>
       </va-card-content>
     </va-card>
+    <PopularCategories :categories="filteredCategories" />
```

Comment:
Implementation is off visually

<img width="886" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/b8e95bd5-c49e-43f3-be32-63e833aecafc">

Reply:
Still off (less off than before :) )

<img width="719" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/5394573/0dfed8d3-231b-4020-ab08-0f6b277a9ba0">